ExtractorUtils/Cognite/DestinationUtils.cs

```diff
@@ -242,6 +240,23 @@ public static void ConfigureSslPolicy(CertificateConfig config)
             {
                 throw new CogniteUtilsException("Cannot configure Builder: Project is not configured");
             }
+
+            string? _tenant = config.IdpAuthentication?.Tenant.TrimToNull();
+            string? _tokenUrl = config.IdpAuthentication?.TokenUrl.TrimToNull();
+
+            if (!String.IsNullOrWhiteSpace(_tenant) && !String.IsNullOrWhiteSpace(_tokenUrl))
```

Comment:
Use `string` over `String`, please.

Reply:
Also, since you already used `TrimToNull` you can just compare it with `null` here: `_tenant is null`.